config/crd/bases/falcon.crowdstrike.com_falcondeployments.yaml

```
@@ -3399,6 +3399,16 @@ spec:
                             default: false
                             description: Enables the use of GKE Autopilot.
                             type: boolean
+                          cleanupAllowListVersion:
+                            description: Version of the GKE AutoPilot Cleanup Daemonset
+                              for troubleshooting purposes
```

```suggestion
                              allow list for troubleshooting purposes
```
The same comment applies to other descriptions also

---

@cs-alphonse Updated.